.github/ISSUE_TEMPLATE/spike.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Spike 
+about: Research, design, investigate, prototype
+title: ''
+labels: ''
+assignees: ''
+---
+
+# Purpose
+
+In order to [goal], [stakeholder] wants [change].
+
+Given above [question/risk/uncertainty], conducting [research/design/investigation/prototyping] is needed to provide factual knowledge on future steps.
+
+[timebox] of effort has been allocated and once compete, findings will be demonstrated and specific future actions will be decided.
+
+## Acceptance Criteria
```

Comment:
Despite what the linter says, these have to be second-level since the rendered template appears in a page that already has an H1.
```suggestion
### Acceptance Criteria
```